src/app/models/propTypes/promo/index.js

```diff
@@ -18,7 +36,7 @@ export const optimoPromoPropTypes = {
   locators: shape({
     optimoUrn: string.isRequired,
   }),
-  summary: string,
+  summary: oneOfType([optimoPromoSummaryPropTypes, string]),
```

Comment:
This usage of oneOfType gives an error locally. 
<img width="840" alt="Screenshot console error" src="https://user-images.githubusercontent.com/3028997/81661910-bee35080-9434-11ea-8c63-c60259173489.png">
I simplified it to this and it no longer errors:
```suggestion
  summary: optimoPromoSummaryPropTypes,
```
I believe we'll always have Optimo promo summaries of this type in future.

Reply:
There are a very small amount of optimo articles (literally 30 or so), that will never be republished to replace the string summary e.g. https://www.bbc.com/afrique/articles/cx80n852v6mo.json . We thought the prop types should reflect the fact that we should be able to render articles in this format. Will we get any issues in prod if prop types don’t allow for string summaries?

Reply:
Okay, in that case then to fix the error, you just need to update optimoPromoSummaryPropTypes so that it has a `shape()` surrounding the object.